### ISSUE TITLE
jfrog rt dotnet restore -> panic: runtime error: invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by Alaric84** [jfrog/jfrog-cli#1230]
I work with a build inside a debian based build container executed in a azure pipeline.

When calling

```
jfrog rt dotnet restore --build-name="$BUILDNAME" --build-number="$BUILDNUMBER"
```
I get
```
buildcontainer    | === Restore packages ===
buildcontainer    | panic: runtime error: invalid memory address or nil pointer dereference
buildcontainer    | [signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0xca077f]
buildcontainer    | 
buildcontainer    | goroutine 1 [running]:
buildcontainer    | github.com/jfrog/jfrog-cli/artifactory.dotnetCmd(0xc000033600, 0xc000196a90, 0x8a3be5)
buildcontainer    | 	/var/jenkins_home/workspace/eco-system/release/jfrog-cli-release/temp/jfrog-cli/artifactory/cli.go:1446 +0x40f
buildcontainer    | github.com/jfrog/jfrog-cli/artifactory.GetCommands.func40(0xc000033600, 0x0, 0xc000033600)
buildcontainer    | 	/var/jenkins_home/workspace/eco-system/release/jfrog-cli-release/temp/jfrog-cli/artifactory/cli.go:628 +0x2b
buildcontainer    | github.com/codegangsta/cli.HandleAction(0xd70f00, 0xf02018, 0xc000033600, 0xc000031100, 0x0)
buildcontainer    | 	/root/go/pkg/mod/github.com/codegangsta/cli@v1.20.0/app.go:490 +0xc8
buildcontainer    | github.com/codegangsta/cli.Command.Run(0xeb87ce, 0x6, 0x0, 0x0, 0x0, 0x0, 0x0, 0xec0408, 0x11, 0xee8713, ...)
buildcontainer    | 	/root/go/pkg/mod/github.com/codegangsta/cli@v1.20.0/command.go:210 +0x9e8
buildcontainer    | github.com/codegangsta/cli.(*App).RunAsSubcommand(0xc00018d380, 0xc000033340, 0x0, 0x0)
buildcontainer    | 	/root/go/pkg/mod/github.com/codegangsta/cli@v1.20.0/app.go:379 +0x88b
buildcontainer    | github.com/codegangsta/cli.Command.startApp(0xeb67e6, 0x2, 0x0, 0x0, 0x0, 0x0, 0x0, 0xec2e2c, 0x14, 0x0, ...)
buildcontainer    | 	/root/go/pkg/mod/github.com/codegangsta/cli@v1.20.0/command.go:298 +0x81a
buildcontainer    | github.com/codegangsta/cli.Command.Run(0xeb67e6, 0x2, 0x0, 0x0, 0x0, 0x0, 0x0, 0xec2e2c, 0x14, 0x0, ...)
buildcontainer    | 	/root/go/pkg/mod/github.com/codegangsta/cli@v1.20.0/command.go:98 +0x1219
buildcontainer    | github.com/codegangsta/cli.(*App).Run(0xc00018d1e0, 0xc0000301e0, 0x6, 0x6, 0x0, 0x0)
buildcontainer    | 	/root/go/pkg/mod/github.com/codegangsta/cli@v1.20.0/app.go:255 +0x741
buildcontainer    | main.execMain(0xc000197f78, 0x405b6f)
buildcontainer    | 	/var/jenkins_home/workspace/eco-system/release/jfrog-cli-release/temp/jfrog-cli/main.go:98 +0x1ab
buildcontainer    | main.main()
```

**Expected behavior**
Execute the restore.

**Versions**
- JFrog CLI version: 1.43.2
- JFrog CLI operating system: Debian Buster
- Artifactory Version: Enterprise Plus 7.19.4 rev

**Additional context**

**Comment by yahavi** [comment]
@Alaric84 
Thanks for reporting this issue. 
Could you please provide the JFrog CLI version used in the task?

**Comment by Alaric84** [comment]
@yahavi sorry - I missed the main point.
I updated the issu. Version printed by the jfrog command is: 1.43.2

**Comment by Alaric84** [comment]
fixed by updating jfrog-cli to 1.51.2
